doc/tutorials/basic/build_report.md

```diff
@@ -0,0 +1,185 @@
+
+# Plot Data
+
+df = df[df.t_manu.isin(top_manufacturers)]
+fig = (
+    alt.Chart(
+        df.sample(5000),
+        title="Capacity by Manufacturer",
+    )
+    .mark_circle(size=8)
+    .encode(
+        y="t_manu:N",
+        x="p_cap:Q",
+        yOffset="jitter:Q",
+        color=alt.Color("t_manu:N").legend(None),
+        tooltip=["t_manu", "p_cap"],
+    )
+    .transform_calculate(jitter="sqrt(-2*log(random()))*cos(2*PI*random())")
+    .properties(
+        height=400,
+        width="container",
+    )
+)
+
+## Build components: display, layout, and style the objects
+
+BRAND_COLOR = "teal"
+BRAND_TEXT_ON_COLOR = "white"
+
+CARD_STYLE = {
+  "box-shadow": "rgba(50, 50, 93, 0.25) 0px 6px 12px -2px, rgba(0, 0, 0, 0.3) 0px 3px 7px -3px",
+  "padding": "10px",
+  "border-radius": "5px"
+}
+
+header = pn.Row(
+    pn.pane.Markdown(
+        "# Wind Turbine Report", styles={"color": BRAND_TEXT_ON_COLOR}, margin=(5, 20)
+    ),
+    styles={"background": BRAND_COLOR},
+)
+indicators = pn.FlexBox(
+    pn.indicators.Number(
+        value=total_capacity / 1e6,
+        name="Total Capacity (TW)",
+        format="{value:,.0f}",
+        styles=CARD_STYLE,
+    ),
+    pn.indicators.Number(
+        value=avg_capacity,
+        name="Avg. Capacity (MW)",
+        format="{value:,.1f}",
+        styles=CARD_STYLE,
+    ),
+    pn.indicators.Number(
+        value=avg_rotor_diameter,
+        name="Avg. Rotor Diameter (m)",
+        format="{value:,.0f}",
+        styles=CARD_STYLE,
+    ),
+    pn.indicators.Number(
+        value=avg_rotor_diameter,
+        name="Avg. Rotor Diameter (m)",
+        format="{value:,.0f}",
+        styles=CARD_STYLE,
+    ),
+    pn.indicators.Number(
+        value=count,
+        name="Count",
+        format="{value:,.0f}",
+        styles=CARD_STYLE,
+    ),
+    margin=(20, 5),
+)
+
+plot = pn.pane.Vega(
+    fig,
+    styles=CARD_STYLE,
+    margin=10,
+)
+
+table = pn.pane.DataFrame(example_df, styles=CARD_STYLE)
+
+## Put the components together
+
+main = pn.Column(
+    "# Summary",
+    indicators,
+    TEXT,
+    "## Manufacturer Capacities",
+    plot,
+    "## Turbine Examples",
+    table,
+)
+
+main_container = pn.Row(
+    main,
+    max_width=1024,
+    styles={"margin-right": "auto", "margin-left": "auto", "margin-top": "10px", "margin-bottom": "20px"},
+)
+report = pn.Column(header, main_container, sizing_mode="stretch_width")
+
+## Export and save it
+
+report.save("report.html")
+
+report.servable()
+```
+
+:::{note}
+If the code was run in a notebook or Python file, the report will have been saved by `report.save("report.html")` to the file `report.html`.
+:::
+
+Please verify that the file `report.html` has been created.
+
+## Distribute the report
+
+Attach the `report.html` file generated in the previous section to an email and send it to yourself. When you receive the email, then open the `report.html` attachment.
```

Comment:
I don't think its weird. But it might be surpricing because the instruction goes outside doing something with Panel.

We actually help a basic user get over 2 barriers: 1) Creating a report 2) Distributing a report.

When you have tried something in practice its easier to do later.

For me the purpose is not to help my users create Panel apps. The purpose is to help my users create value in their job using Panel. I want my colleagues to be able to distribute reports. Not just to create them.

Reply:
Closing. Feel free to reopen if you disagree strongly.

Reply:
I don't think users will actually send an email to themselves even if you ask them to do so. I wouldn't!